### ISSUE TITLE
Suppress expected AMAPIError messages

### ISSUE THREAD

**Issue by ahelsing** [GENI-NSF/geni-tools#187]
The acceptance tests try to create error conditions.
The result is that aggregates respond with AM API error codes (non-0).
Omni in turn raises an Exception: an AMAPIError.

The acceptance tests could suppress those messages that are expected, or otherwise make clear they are expected.

 _Imported from trac ticket [#187](http://trac.gpolab.bbn.com/gcf/ticket/187),  created by ahelsing on 09-06-2012 at 10:42, last modified: 09-13-2012 at 13:51_
   CCing: lnevers@...

**Comment by ahelsing** [comment]
the  AM API acceptance test for PLC using gcf 2.0 rc3:

```
lnevers@arendia:~/gcf-2.0-rc3/acceptance_tests/AM_API$ ./am_api_accept.py -a am-undertest --more-strict -V2
FOmni raised an error: AMAPIError: Error from Aggregate: code 2. sfa AM code: 2: : SliverStatus: Sliver does not exist : pgeni\.gpolab\.bbn\.com.non-140335 (used pgenigpolabbbncom_non140335 as slicename internally).
FFOmni raised an error: AMAPIError: Error from Aggregate: code 2. sfa AM code: 2: : CreateSliver: Invalid RSpec: No RSpec or version specified. Must specify a valid rspec string or a valid version.
.Omni raised an error: AMAPIError: Error from Aggregate: code 2. sfa AM code: 2: : CreateSliver: Invalid XML Document: Premature end of data in tag rspec line 35, line 36, column 1.
...Omni raised an error: AMAPIError: Error from Aggregate: code 3. sfa AM code: 3: : ListResources: Insufficient rights: Access denied: <class 'sfa.util.faults.CredentialNotVerifiable'> -- u"[ Grant [ OU: bbn-pgeni.lnevers, CN: 23c6477d-d231-11df-92ee-000c29f89f7b, SubjectAltName: URI:urn:publicid:IDN+pgeni.gpolab.bbn.com+user+lnevers, email:lnevers@pgeni.gpolab.bbn.com ] rights on [ OU: bbn-pgeni.sa, CN: 8fe74255-6a96-11df-9d02-000c29f89f7b, SubjectAltName: URI:urn:publicid:IDN+pgeni.gpolab.bbn.com+authority+sa ] until 2012-09-07 13:45:19 ]: Element 'foo': This element is not expected. Expected is ( credential ). (line 2)".
..Omni raised an error: AMAPIError: Error from Aggregate: code 5. sfa AM code: 5: Internal API error: not well-formed (invalid token): line 1, column 0.
...Omni raised an error: AMAPIError: Error from Aggregate: code 3. sfa AM code: 3: : ListResources: Insufficient rights: Access denied: <class 'sfa.util.faults.ConnectionKeyGIDMismatch'> -- u'geni//gpo//gcf.user.alice:23c6477d-d231-11df-92ee-000c29f89f7b'. 
```


_Trac comment by ahelsing on 09-06-2012 at 10:43_

**Comment by seledwards** [comment]
We fixed this in 2.0.

Reopen if we find it again.

_Trac comment by sedwards (github user: seledwards) on 09-13-2012 at 13:51_
